indra/sources/crog/processor.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+"""Processor for the `Chemical Roles Graph (CRoG)
+<https://github.com/chemical-roles/chemical-roles>`_.
+"""
+
+from ..utils import RemoteProcessor
+
+__all__ = [
+    'CrogProcessor',
+]
+
+CROG_URL = 'https://raw.githubusercontent.com/chemical-roles/' \
+           'chemical-roles/master/docs/_data/crog.indra.json'
+
+
+class CrogProcessor(RemoteProcessor):
```

Comment:
I would actually still add an `api.py` that has an edpoint like `process_from_web`. That would fit with the behavior of other sources.

Reply:
okay will do, I will make it possible to give an optional URL to override the internal one (perhaps for testing purposes)

Reply:
Addressed in 4a247d1da